### ISSUE TITLE
windows - vim mode - default binding does not work for azerty

### ISSUE THREAD

**Issue by IzioDev** [zed-industries/zed#14860]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

This is not really a "bug" but rather a default configuration that could be improved, i think.

On Zed, on Windows, with an azerty keyboard, with vim mode enabled, i am not able to do the `|` character (alt + -)

it looke like it does a `Control + -` Go back even when not in insert mode

interestingly, doing `Alt + _` (assuming it would do a `Control + _`) does not do a Go forward

### Environment

```
Zed: v0.145.0 (Zed Dev ca80343486a77903327bf413a14faf119bdf05ae)
OS: Windows 10.0.22631
Memory: 63.1 GiB
Architecture: x86_64
```

### If applicable, add mockups / screenshots to help explain present your vision of the feature

![image](https://github.com/user-attachments/assets/bdb80713-cd60-4a68-8df5-821b1e6db9fe)

![image](https://github.com/user-attachments/assets/24b27f6d-db69-4e0e-8295-3a8246cab172)
: it has inserted a pipe but after a move cursor, which seems weird

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>

</pre></details>

**Comment by notpeter** [comment]
Interesting. I don't see anything mapped to `alt--` but `ctrl--` and `ctrl-alt--` are mapped:

https://github.com/zed-industries/zed/blob/f31c55a76f73824e4f6a9a983610a67115982223/assets/keymaps/default-linux.json#L28-L29

https://github.com/zed-industries/zed/blob/f31c55a76f73824e4f6a9a983610a67115982223/assets/keymaps/default-linux.json#L325-L326

**Comment by notpeter** [comment]
Yep. Windows uses the same keymap as Linux, so this is the right file.
